flink-connectors/flink-connector-kafka/pom.xml

```diff
@@ -140,28 +140,6 @@ under the License.
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-connector-kafka-base_${scala.binary.version}</artifactId>
 			<version>${project.version}</version>
-			<exclusions>
-				<!-- exclude Kafka dependencies -->
-				<exclusion>
-					<groupId>org.apache.kafka</groupId>
-					<artifactId>kafka_${scala.binary.version}</artifactId>
```

Comment:
the exclusion is still useful for dependency convergence.

Reply:
Are transitive test dependencies considered for dependency convergence checks?

Reply:
yes, since they are still available during tests which dependency convergence covers as well.

Reply:
I think testing dependencies don't pull in transitive dependencies: https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html.

Reply:
if kafka is just a test dependency of flink-kafka-connector-base, then yes it is irrelevant for dependency convergence.